roles/convert2rhel/defaults/main.yml

```diff
@@ -16,3 +17,7 @@ foreman_convert2rhel_content_view: "Default Organization View"
 # Host groups
 foreman_convert2rhel_hostgroup7: "CentOS 7 converting"
 foreman_convert2rhel_hostgroup8: "CentOS 8 converting"
+foreman_convert2rhel_hg_ol7: "Oracle 7 converting"
```

Comment:
should this be `foreman_convert2rhel_hostgroup_ol7` to match the others?

or should the others also end in `_centosX`?